aixplain/model_interfaces/interfaces/function_models.py

```diff
@@ -179,3 +183,52 @@ def predict(self, request: Dict[str, str], headers: Dict[str, str] = None) -> Di
             TextToImageGenerationOutput(**text_to_image_generation_dict)
             text_to_image_generation_output["predictions"][i] = text_to_image_generation_dict
         return text_to_image_generation_output
+    
+class TextGenerationModel(AixplainModel):
+    def run_model(self, api_input: Dict[str, List[TextGenerationInput]], headers: Dict[str, str] = None) -> Dict[str, List[TextGenerationOutput]]:
```

Comment:
api_input should be `Dict[str, List[TextInput]]`, the same as in TextGenerationChatModel